src/scipp/coords/transform_coords.py

```diff
@@ -14,14 +15,16 @@
 
 
 def transform_coords(x: Union[DataArray, Dataset],
-                     targets: Union[str, Iterable[str]],
-                     graph: GraphDict,
+                     targets: Optional[Union[str, Iterable[str]]] = None,
```

Comment:
It's a small thing but should we list in the breaking changes that `targets` is not position only?

Reply:
Done